app/controllers/TEAHubController.scala

```diff
@@ -9,16 +10,33 @@ import scala.concurrent.{ExecutionContext, Future}
 /** Is a controller for the project
   *
   * @param togglService Service that requests Toggl the list of the projects */
-class TEAHubController(togglService: TogglService)(implicit executionContext: ExecutionContext) extends Controller {
+class TEAHubController(togglService: TogglService, cache: CacheApi)(implicit executionContext: ExecutionContext) extends Controller {
```

Comment:
I don't understand why are we adding Cache, can you provide details in the commit message ?